gen_cert.py

```diff
@@ -510,7 +534,9 @@ def _generate_certificate(
                         designation=designation,
                     )
                 achievements_string = value['credits']
-                achievements_description_string = unicode(self.cert_data['CREDITS'])
+                achievements_description_string = unicode(
+                    self.cert_data.get('CREDITS', '')
```

Comment:
I don't think they must be, but there is a relationship between designation and credits that will probably be dealt with when sub-templates fully replaces the designations.